department-of-veterans-affairs/caseflow
# Updated - Khalin/appeals 42565

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-42565)

# Description
 Record Status now updates to "Completed" when "Tasks not related to an appeal" section is empty. Banner also now references "Other Motion". 

## Acceptance Criteria

## Testing Plan

1. Log into Caseflow as a supervisor user
2. Go to "Your Queue"
3. Click "Correspondence Cases" in the "Switch Views" drop down
4. Click any veteran in the "Veteran Details" column
5. Select any Correspondence Type and click Save Changes
6. Scroll to the bottom and click "Create Record"
7. Click Continue to go to the second step
8. Under the "Tasks not related to an Appeal" Click "+Add Task"
9. In the New Task modal that renders add and "Other Motion" task and add any test text in the instructions
10. Scroll to the bottom and click "Continue"
11. Scroll to the bottom and click "Submit" then "Confirm" in the pop-up modal
12. Copy the name that shows up in the support banner on the next redirected page(/correspondence/team)
13. Go to the "Pending" tab
14. Paste the name that you copied from the success banner
15. Click the result with "Other motion" under the "Tasks" section
16. Copy the URL 
17. Log into Caseflow as a LIT SUPPORT USER
18. Paste the copied URL
19. Scroll to the bottom click the actions dropdown in the "Tasks not related to an appeal" section
20. Click Cancel task 
21. Add any text in the instructions checkbox 
22. Click "Cancel Task"
23. Check that the success banner renders as intended


# Frontend
## User Facing Changes
<img width="1251" alt="Screenshot 2024-08-21 at 10 51 10 AM" src="https://github.com/user-attachments/assets/efc832fc-ef8b-48b2-a335-e16a3738439d">
